client/package.json

```diff
@@ -28,6 +28,9 @@
   },
   "dependencies": {
     "final-form": "^4.11.0",
-    "react-final-form": "^4.0.2"
+    "react-final-form": "^4.0.2",
+    "react-responsive": "^6.0.1",
+    "react-select": "^2.1.2",
+    "styled-components": "^4.1.2"
```

Comment:
nice

Reply:
we should use css modules

Reply:
I've seen people have success with css modules but I wanted to try out `styled-components` for this project